CHANGES.rst

```diff
@@ -118,6 +118,9 @@ New Features
 
   - Allow to use a tuple of keys in ``Table.sort``.  [#4671]
 
+  - Added interface to create ``Table`` directly from any "compatible" table
```

Comment:
Maybe point out that the definition of "compatible" is described in more detail in the docs?  At first I was like "what does compatible mean?  and why is it in quotes?"